packages/ast-spec/src/declaration/TSModuleDeclaration/fixtures/namespace-nested-once-declare/snapshots/1-TSESTree-AST.shot

```diff
@@ -0,0 +1,71 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AST Fixtures declaration TSModuleDeclaration namespace-nested-once-declare TSESTree - AST 1`] = `
+Program {
+  type: "Program",
+  body: [
+    TSModuleDeclaration {
+      type: "TSModuleDeclaration",
+      body: TSModuleBlock {
+        type: "TSModuleBlock",
+        body: [],
+
+        range: [26, 28],
+        loc: {
+          start: { column: 26, line: 1 },
+          end: { column: 28, line: 1 },
+        },
+      },
+      declare: false,
```

Comment:
your fix has not worked.

Reply:
Thank you. I've fixed [f43949d](https://github.com/typescript-eslint/typescript-eslint/pull/7410/commits/f43949de5a55bb6ab96878d496dac5eeb88c9638) and updated snapshots [e8463d2](https://github.com/typescript-eslint/typescript-eslint/pull/7410/commits/e8463d276b42d8621c1f56647b3abf6006f0c806)